cmd/infracost/run_test.go

```diff
@@ -64,3 +65,18 @@ func TestCatchesRuntimeError(t *testing.T) {
 		c.Config.Projects = []*config.Project{nil, nil}
 	})
 }
+
+func TestConfigFileWithYorConfig(t *testing.T) {
+	GoldenFileCommandTest(t, testutil.CalcGoldenFileTestdataDirName(), []string{"breakdown", "--config-file", "./testdata/infracost-config-yor.yml", "--format", "json"}, &GoldenFileOptions{
+		IsJSON:      true,
+		RegexFilter: regexp.MustCompile(`"path": "[^"]*",`),
+	})
+}
+
+func TestConfigFileWithYorEnv(t *testing.T) {
+	t.Setenv("YOR_SIMPLE_TAGS", `{"Team": "Engineering"}`)
+	GoldenFileCommandTest(t, testutil.CalcGoldenFileTestdataDirName(), []string{"breakdown", "--config-file", "./testdata/infracost-config-yor.yml", "--format", "json"}, &GoldenFileOptions{
+		IsJSON:      true,
+		RegexFilter: regexp.MustCompile(`"path": "[^"]*",`),
```

Comment:
I'm curious why the 
```
JSONInclude: regexp.MustCompile("^(defaultTags|tags|name)$"),
JSONExclude: regexp.MustCompile("^(costComponents|pastBreakdown)$"),
``` 
didn't work for you? It would think it made the golden files a lost less noisy?

Reply:
My main problem was misunderstanding how they worked tbh, I think they would work if I listed needed properties in the include - I had tried `.*` in include and `path` in exclude which failed to exclude path because an ancestor had already been included.

Reply:
I guess it's "Show me only keys that match JSONInclude anywhere they appear in the tree, unless they appear under a key that matches JSONExclude"